test/util/test_path_provider.py

```diff
@@ -0,0 +1,172 @@
+import os
+from pathlib import Path
+from abc import ABC, abstractmethod
+
+
+class TestPathProviderBase(ABC):
+    """
+    used to generate the files path's in all tests
```

Comment:
my comment disappeared again :( please align these comments so they are even

...
...

not 
........
..
....
.